### PR TITLE
Minimal implementation to add tooltips to table view from meta data;

### LINKED ISSUE #383: Missing metadata tooltips for each column
As a   user, I would like to have a metadata entry displayed for each column, so   that I know which of my data correspond to which column.

### PR DESCRIPTION
**Pull request by wingechr**
Fixes #383 

TODOs:

- [ ] get metadata via API
- [ ] pretty html
- [ ] extended description (units etc)

### CODE REVIEW

**Comment by jh-RLI** [comment]
Can you refer to the issues that will be closed with this PR? 
Also this could be [helpfull](https://help.github.com/en/articles/closing-issues-using-keywords).

**Comment by Bachibouzouk** [comment]
Closes #383

File: dataedit/templates/dataedit/dataedit_overview.html [hunk 1]
```diff
@@ -109,6 +110,29 @@ <h3><a href="/dataedit/view/{{schema}}">{{schema}}</a>.{{table}} </h3>
...
+            var colDiv = jQuery(e);
+            var colName = colDiv.text();
+            var tooltipHTML = colname2tooltip[colName];
+            // TODO: html does not work
+            colDiv.attr('data-html', "true").attr('data-toggle', 'tooltip').attr('title', tooltipHTML);
+        });
+    };
+
+
     function build_views(){
```
Comment (Bachibouzouk) [inline dataedit/templates/dataedit/dataedit_overview.html]: @wingechr - This function has apparently been replaced by `build_views_leaflet` and `build_views_recline`. I am not sure how to solve the merge conflict here

**Comment by jh-RLI** [comment]
@wingechr @Bachibouzouk can you verify that this PR also closes #297 ?

**Comment by christian-rli** [comment]
> @wingechr @Bachibouzouk can you verify that this PR also closes #297 ?

I can confirm.

**Comment by christian-rli** [comment]
Closes #297

**Comment by Bachibouzouk** [comment]
This is now implemented in #430

**Pull request closed without merging**